charts/llm-d-modelservice/values.yaml

```diff
@@ -271,7 +273,7 @@ prefill:
       # enabled -- Create PodMonitor resource for prefill deployment
       enabled: false
       # port -- Port name or number to scrape metrics from (defaults to vllmPort: decode=8200, prefill=8000)
-      # port: 8000
+      # port: "metrics"
```

Comment:
I wonder if `portName` is a better word choice here than port if the intention to to use a port name rather than a port number?

Reply:
in a follow-up we can add an option for `portNumber`, also.